### ISSUE TITLE
Add request analysisd configuration test

### ISSUE THREAD

**Issue by juliamagan** [wazuh/wazuh-qa#3604]
| Target version | Related issue | 
|--------------------|--------------------|
|            4.4            |           #3112             |           

## Description
We need to add a test that checks if the analysisd configuration, requested with the API is the expected.

## Proposed test cases

- [x] Get and check the default configuration
- [x] Get and check custom configuration

**Comment by juliamagan** [comment]
## Update 15/12/2022

Try to keep the test as simple as possible and reuse the one implemented in the previous issue.

Half of the endpoints remain.

**Comment by juliamagan** [comment]
## Update 23/12/2022

This test must be merged after #3603. It remains to apply the changes that were applied to #3603 on the expected responses so that it follows the same format.

In addition, this test is blocked by:

- https://github.com/wazuh/wazuh/issues/15716
- https://github.com/wazuh/wazuh/issues/15725
- https://github.com/wazuh/wazuh/issues/15733

The following `internal_options` are missing in the API response:

- `analysisd.geoip_jsonout`
- `analysisd.event_threads`
- `analysisd.syscheck_threads`
- `analysisd.syscollector_threads`
- `analysisd.rootcheck_threads`
- `analysisd.sca_threads`
- `analysisd.hostinfo_threads`
- `analysisd.rule_matching_threads`
- `analysisd.winevt_threads`
- `analysisd.decode_event_queue_size`
- `analysisd.decode_syscheck_queue_size`
- `analysisd.decode_syscollector_queue_size`
- `analysisd.decode_rootcheck_queue_size`
- `analysisd.decode_sca_queue_size`
- `analysisd.decode_hostinfo_queue_size`
- `analysisd.decode_output_queue_size`
- `analysisd.decode_winevt_queue_size`
- `analysisd.archives_queue_size`
- `analysisd.statistical_queue_size`
- `analysisd.alerts_queue_size`
- `analysisd.firewall_queue_size`
- `analysisd.fts_queue_size`
- `analysisd.dbsync_queue_size`
- `analysisd.upgrade_queue_size`
- `analysisd.state_interval`

Could be related to:
- https://github.com/wazuh/wazuh/issues/15694

**Comment by vikman90** [comment]
Closing this issue. Superseded by version 5.0.
